app/models/early_career_payments.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# Module namespace specific to the policy for claiming early career payments.
+#
+# TODO: Put in details of what this is for.
+module EarlyCareerPayments
+  extend self
+
+  POLICY_START_YEAR = AcademicYear.new(2021).freeze
+
+  def start_page_url
+    if Rails.env.production?
+      "https://www.gov.uk/guidance/early-career-payments-guidance-for-teachers-and-schools"
+    else
+      "/#{routing_name}/claim"
+    end
+  end
+
+  def eligibility_page_url
+    "https://www.gov.uk/publications/TO-BE-REPLACED-by-response-to-ECP-518"
```

Comment:
This can't be done now?

Reply:
@nomasprime This was required in order to get the skeleton up and running as we progressed through BDD/TDD cycles. As the publication is not yet published (there are ones for M+P / TSLR at the link https://www.gov.uk/guidance/early-career-payments-guidance-for-teachers-and-schools
but there is no entry yet for the new Early Career Payments journey. Yes this would break if a user follows the link. However by the time we deliver to Private Beta I expect the page to be available and its associated link

Reply:
Maybe this should have been as a TODO?